### DISCUSSION TITLE
[Enhancement] [select] add popper-options attribute

### DISCUSSION THREAD

**Discussion by yyq-bit** [element-plus/element-plus#11553] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `select`

## Description

- Can select add popper-options attribute, like select-v2?

  I want to hide the arrow, and reset the distance between select-dropdown and input.

  Any solutions except setting popper-options attribute and hiding the arrow via css?

  ```vue
  <script setup>
  const popperOptions = {
    modifiers: [{ name: 'offset', options: { offset: [0, 0] } }],
  }
  </script>
  <template>
    <el-select :popper-options="popperOptions">
      ...
    </el-select>
  </template>
  ```

### Additional comments

If this is possible, I can submit a pr.

**Comment by nailfar** [comment]
Also date-picker poper ...all floating ui used popper.js
